src/nvme/util.h

```diff
@@ -150,6 +150,40 @@ void nvme_init_copy_range_f1(struct nvme_copy_range_f1 *copy, __u16 *nlbs,
 			     __u64 *slbas, __u64 *eilbrts, __u32 *elbatms,
 			     __u32 *elbats, __u16 nr);
 
+/**
+ * nvme_init_copy_range_f2() - Constructs a copy range f2 structure
+ * @copy:	Copy range array
+ * @snsids:	Source namespace identifier
+ * @nlbs:	Number of logical blocks
+ * @slbas:	Starting LBA
+ * @sopts:	Source options
+ * @eilbrts:	Expected initial logical block reference tag
+ * @elbatms:	Expected logical block application tag mask
+ * @elbats:	Expected logical block application tag
+ * @nr:		Number of descriptors to construct
+ */
+void nvme_init_copy_range_f2(struct nvme_copy_range_f2 *copy, __u32 *snsids,
+			     __u16 *nlbs, __u64 *slbas, __u16 *sopts,
+			     __u32 *eilbrts, __u32 *elbatms, __u32 *elbats,
```

Comment:
All pointers besides `struct nvme_copy_range_f2 *copy` could be marked `const`

Reply:
Another change? The existing nvme_init_copy_range and nvme_init_copy_range_f1 also have this issue.

Reply:
The existing functions could be modified too; I think that's a backwards-compatible change. But if you'd rather not do that, I'm fine with dropping the comment.

Reply:
This requires a corresponding change in nvme-cli's `libnvme-wrap.c`. If we expect nvme-cli and libnvme to always be "in sync" I think it's ok to make this change, but I'm not sure if that's the case.

Reply:
Yes, they are distributed together, but can be built separately. That's a fair point, so let's not worry about making things `const`.